src/event_loop.rs

```diff
@@ -154,6 +154,14 @@ impl<T> EventLoop<T> {
         self.event_loop.run(event_handler)
     }
 
+    #[inline]
+    pub fn run_return<F>(&mut self, event_handler: F)
```

Comment:
`run_return` is already available on Android with the `platform::run_return::EventLoopExtRunReturn` trait

Reply:
I will check this again! Maybe it can be removed.

Reply:
I removed this.

Reply:
Don't forget to remove it from the changelog too.

While at it, make sure crate users understand what changes to make, or how they can take advantage of the new functionality that is added.